benchmarks/microbenchmarks/memcpy.py

```diff
@@ -0,0 +1,36 @@
+from microbenchmarks._items import BenchmarkItem, Container, DataSize, DataType
+from microbenchmarks._metric import MetricType
+from microbenchmarks._plan import BenchmarkPlan
+from microbenchmarks._utils import dtype_size, scaled_repeat_times
```

Comment:
fill some random data will be better
```suggestion
from microbenchmarks._utils import dtype_size, fill_random, scaled_repeat_times
```